modules/tinymce/src/themes/silver/test/ts/webdriver/editor/EventsTest.ts

```diff
@@ -0,0 +1,90 @@
+import { Mouse, UiFinder, Waiter } from '@ephox/agar';
+import { before, describe, it } from '@ephox/bedrock-client';
+import { McEditor, TinyContentActions, TinyDom, TinyUiActions } from '@ephox/mcagar';
+import { SugarBody } from '@ephox/sugar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('webdriver.tinymce.themes.silver.editor.EventsTest', () => {
+  const settings = {
+    base_url: '/project/tinymce/js/tinymce',
+    plugins: 'link',
+    menubar: 'file edit'
+  };
+
+  const contextMenuItemSelector = '[role="menuitem"]:contains("Link...")';
+  const fileMenuSelector = '[role="menuitem"]:contains("New document")';
+  const editMenuSelector = '[role="menuitem"]:contains("Cut")';
+  const fileMenuButtonSelector = 'button:contains("File")';
+  const editMenuButtonSelector = 'button:contains("Edit")';
+
+  before(() => {
+    Theme();
+  });
```

Comment:
I'd do a one line for this:
```
before(() => Theme());
```

Reply:
We can't do shorthand hooks because the result from `Theme()` is returned but it really should be `void`

I fixed others here https://github.com/tinymce/tinymce/pull/6921